smithy-syntax/src/main/java/software/amazon/smithy/syntax/Formatter.java

```diff
@@ -427,9 +443,18 @@ private Doc visit(TreeCursor cursor) {
                            : Doc.text(tree.concatTokens());
                 }
 
+                case TEXT_BLOCK: {
+                    // Dispersing the lines of the text block preserves any indentation applied from formatting parent
+                    // nodes.
+                    List<Doc> lines = Arrays.stream(tree.concatTokens()
+                            .split(System.lineSeparator()))
```

Comment:
This belongs on the line above, as it's an operation on the result of `concatTokens()` and not on the result of `Arrays.stream()`.